docs/querying/functions.md

```diff
@@ -847,6 +847,14 @@ additional functions are available:
 
 * `mad_over_time(range-vector)`: the median absolute deviation of all float
   samples in the specified interval.
+* `tmin_over_time(range-vector)`: the unix timestamp of minimum value of its
+  argument. If there are multiple, the timestamp of the last occurance is the
+  result.
```

Comment:
We generally capitalize Unix. However, in the context of PromQL, the timestamp is a sufficiently clear concept already. I would leave out the "Unix" (cf. documentation of `timestamp`).

With same other word smithing, I would propose:

```suggestion
* `tmin_over_time(range-vector)`: the timestamp of the last float sample that has the minimum value of all float samples in the specified interval.
```

(And similar changes for the others.)

Reply:
:accept: